.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Python Script
+
+on:
+  push:
+    branches:
+      - rapid-inflows
```

Comment:
i don't think we are supposed to list specific branches. we need to double check this part.

Reply:
You're right, this was just provided to me in the template they create. Updated